aggregator/options.go

```diff
@@ -612,7 +656,9 @@ func (o *options) computeFullGaugePrefix() {
 
 // By default we use e.g. ".p50", ".p95", ".p99" for the 50th/95th/99th percentile.
 func defaultTimerQuantileSuffixFn(quantile float64) []byte {
-	return []byte(fmt.Sprintf(".p%0.0f", quantile*100))
+	s := fmt.Sprintf(".p%0.0f", quantile*10000)
```

Comment:
This makes assumptions about what quantiles we'll support, can we instead write a function to keep multiplying the quantile by 10 until it becomes an integer and use that in the string?